regipy_tests/plugin_tests.py

```diff
@@ -1,5 +1,5 @@
 import pytest
-
+import datetime
```

Comment:
datetime is usually (as a convention) imported as:
```
import datetime as dt
```

and then used like this: 
```
 dt.datetime(2021, 8, 16, 9, 43, 22).isoformat()
```